lua/astrocommunity/test/neotest/init.lua

```diff
@@ -8,6 +8,7 @@ return {
     {
       "folke/neodev.nvim",
       opts = function(_, opts)
+        opts.library = opts.library or {}
         opts.library.plugins = require("astronvim.utils").list_insert_unique(opts.library.plugins, "neotest")
```

Comment:
```suggestion
        if opts.library.plugins ~= true then
          opts.library.plugins = require("astronvim.utils").list_insert_unique(opts.library.plugins, "neotest")
        end
```

Reply:
Docs for reference: https://github.com/folke/neodev.nvim#%EF%B8%8F-configuration